addons/l10n_us/models/res_config_settings.py

```diff
@@ -0,0 +1,9 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    module_account_taxcloud = fields.Boolean(string="Account TaxCloud")
```

Comment:
Perhaps the field should be renamed to `module_l10n_us_taxcloud` and the string too ?

Reply:
I will ask Olma about the string, the name of the field could be changed yes !

Reply:
Alrighty, let me know so I can send the PR to final reviewer if it's OK like this :)

Reply:
Yesss thanks for the review !

Reply:
Oh forgot to say, it'd be good to link the Enterprise and Upgrade PRs in the PR description (on all PRs)

Reply:
I have to change the commit tilte aswell, will do that thanks :smile:

Reply:
Actually i don't think the sting is displayed for users so let's changed it also :smile: